multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -24,20 +31,672 @@ use node_lib::database::db_initializer::{
     DbInitializationConfig, DbInitializer, DbInitializerReal, ExternalData,
 };
 use node_lib::sub_lib::accountant::PaymentThresholds;
+use node_lib::sub_lib::blockchain_interface_web3::{
+    compute_gas_limit, transaction_data_web3, web3_gas_limit_const_part,
+};
 use node_lib::sub_lib::wallet::Wallet;
 use node_lib::test_utils;
+use rusqlite::ToSql;
 use rustc_hex::{FromHex, ToHex};
 use std::convert::TryFrom;
 use std::path::{Path, PathBuf};
 use std::time::{Duration, Instant, SystemTime};
 use std::{thread, u128};
 use tiny_hderive::bip32::ExtendedPrivKey;
 use web3::transports::Http;
-use web3::types::{Address, Bytes, TransactionParameters};
+use web3::types::{Address, Bytes, SignedTransaction, TransactionParameters, TransactionRequest};
 use web3::Web3;
 
 #[test]
 fn verify_bill_payment() {
+    // Note: besides the main objectives of this test, it relies on (and so it proves) the premise
+    // that each Node, after it reaches its full connectivity and becomes able to make a route,
+    // activates its accountancy module whereas it also unleashes the first cycle of the scanners
+    // immediately. That's why some consideration has been made not to take out the passage with
+    // the intense startups of a bunch of Nodes, with that particular reason to fulfill the above
+    // depicted scenario, even though this test could be written more simply with the use of
+    // the `scans` command emitted from a UI, with a smaller PCU burden. (You may want to know that
+    // such an approach is implemented for another test in this file.)
+    let payment_thresholds = PaymentThresholds {
+        threshold_interval_sec: 2_500_000,
+        debt_threshold_gwei: 1_000_000_000,
+        payment_grace_period_sec: 85_000,
+        maturity_threshold_sec: 85_000,
+        permanent_debt_allowed_gwei: 10_000_000,
+        unban_below_gwei: 10_000_000,
+    };
+    let owed_to_serving_node_1_minor =
+        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei) + 123_456;
+    let owed_to_serving_node_2_minor =
+        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei) + 456_789;
+    let owed_to_serving_node_3_minor =
+        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei) + 789_012;
+    let cons_node_initial_service_fee_balance_minor =
+        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei) * 4;
+    let exp_final_cons_node_service_fee_balance_minor = cons_node_initial_service_fee_balance_minor
```

Comment:
I sometimes use `exp` for exponent and you'd prefer it for `expected`. I'd like if you use `expected` to help my silly mind.